src/main/java/ch/njol/skript/lang/Statement.java

```diff
@@ -20,6 +20,7 @@
 
 import ch.njol.skript.Skript;
 import ch.njol.skript.lang.function.EffFunctionCall;
+import ch.njol.skript.lang.parser.ParserInstance;
```

Comment:
```suggestion
```
Doesn't seem to be used

Reply:
<img width="768" alt="image" src="https://github.com/user-attachments/assets/b28ad8ee-08fe-458c-bf53-597877030e84" />